apps/web/pages/auth/login.tsx

```diff
@@ -162,6 +176,7 @@ export default function Login({
               {twoFactorRequired && <TwoFactor center />}
 
               {errorMessage && <Alert severity="error" title={errorMessage} />}
+              {successMessage && <Alert severity="info" title={successMessage} />}
```

Comment:
For some reason the severity "success" was not what I expected in terms of color and style, so using "info" for now.